src/Twig/Component/Menu/TreeComponent.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Menu plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusMenuPlugin\Twig\Component\Menu;
+
+use Doctrine\ORM\EntityManagerInterface;
+use MonsieurBiz\SyliusMenuPlugin\DataProvider\Tree\MenuItemTreeProviderInterface;
+use MonsieurBiz\SyliusMenuPlugin\Entity\MenuInterface;
+use MonsieurBiz\SyliusMenuPlugin\Entity\MenuItemInterface;
+use MonsieurBiz\SyliusMenuPlugin\Manager\MenuPositionHandler;
+use MonsieurBiz\SyliusMenuPlugin\Repository\MenuItemRepository;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveAction;
+use Symfony\UX\LiveComponent\Attribute\LiveArg;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent(name: 'monsieur_biz:menu:tree', template: '@MonsieurBizSyliusMenuPlugin/admin/menu/tree.html.twig')]
```

Comment:
```suggestion
#[AsLiveComponent(name: 'monsieurbiz:menu:tree', template: '@MonsieurBizSyliusMenuPlugin/admin/menu/tree.html.twig')]
```

And anywhere it's called